recipe/meta.yaml

```diff
@@ -1,15 +1,13 @@
 {% set name = "apache-beam" %}
-{% set version = "2.46.0" %}
+{% set version = "2.48.0" %}
 
 package:
   name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
-  sha256: 871e88fb137341f0f3d647ba36938b792db135cb08cfb0664c5124b81437c712
-  patches:
-    - 0001-Relax-dependencies.patch
+  sha256: 611b9e0015e9d1d2ca34b91453117ee5b54ca7446de505b95b2c5a5c4d9f4b1e 
```

Comment:
Do we no longer need the 0001-Relax-dependencies.patch?

Reply:
yes, dependencies are already relaxed in the source code.

Reply:
great.